src/otx/algo/detection/yolox.py

```diff
@@ -115,6 +116,8 @@ def _create_model(self) -> nn.Module:
         return detector
 
     def _customize_inputs(self, entity: DetBatchDataEntity) -> dict[str, Any]:
+        if isinstance(entity.images, list):
+            entity.images = stack_batch(entity.images, pad_size_divisor=32, pad_value=114)
```

Comment:
Shouldn't we have `pad_size_divisor` and `pad_value` in a recipe for the configuration?

Reply:
That value in recipe is for det data preprocessor which is implemented by mmdet. We are now decoupling it and data preprocessor is divided into model and data pipeline. So we now don't take those values from recipe.